README.md

```diff
@@ -54,9 +54,9 @@ spectral lint petstore.yaml
 
 - [Documentation](https://meta.stoplight.io/docs/spectral/docs/getting-started/1-concepts.md)
   - [Getting Started](https://meta.stoplight.io/docs/spectral/docs/getting-started/1-concepts.md) - The basics of Spectral.
-  - [Different Workflows](https://meta.stoplight.io/docs/spectral/docs/guides/1-workflows.md) - When and where should you use Spectral? Editors, Git-hooks, Continuous Integration, GitHub Actions, wherever you like!
+  - [Different Workflows](https://meta.stoplight.io/docs/spectral/docs/getting-started/1-concepts.md) - When and where should you use Spectral? Editors, Git-hooks, Continuous Integration, GitHub Actions, wherever you like!
   - [Using the command-line interface](https://meta.stoplight.io/docs/spectral/docs/guides/2-cli.md) - Quickest way to get going with Spectral is in the CLI.
-  - [Using the JavaScript API](https://meta.stoplight.io/docs/spectral/docs/guides/3-javascript.md) - Access the _raw power_ of Spectral via the JS, or hey, TypeScript if you want.
+  - [Using the JavaScript API](https://meta.stoplight.io/docs/spectral/docs/guides/2a-javascript.md) - Access the _raw power_ of Spectral via the JS, or hey, TypeScript if you want.
```

Comment:
is there any reason why CLI is 2, and JS API 2a?
They are somewhat on par. You usually use either CLI or JS, the understanding of CLI is not needed & helpful if you're interested in the programmatic usage of Spectral.